app/assets/javascripts/retro.js

```diff
@@ -23,9 +21,8 @@ $(document).ready(function() {
 
     const minutes = Math.floor(timeNow / (1000 * 60));
     const seconds = Math.floor(timeNow / 1000) % 60;
-
-    let minutesString = minutes < 10 ? "0" + minutes : minutes;
-    let secondsString = seconds < 10 ? "0" + seconds : seconds;
+    const minutesString = minutes < 10 ? "0" + minutes : minutes;
+    const secondsString = seconds < 10 ? "0" + seconds : seconds;
 
     return minutesString + ":" + secondsString;
```

Comment:
One of the best features of ES6 for me is [text interpolation](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals), maybe you can take a look at it!
Sorry that I didn't mention this on the corresponding pull request, but I think It worth it for the future 😄